.github/workflows/release-project.yaml

```
@@ -59,14 +72,14 @@ jobs:
-              "release_name": "${{ github.event.inputs.release_name }}",
-              "automatic_mode": "${{ github.event.inputs.automatic_mode }}",
-              "prerelease": "${{ github.event.inputs.prerelease }}"
+              "version": "${{ inputs.version }}",
+              "release_name": "${{ inputs.release_name }}",
+              "automatic_mode": "${{ inputs.automatic_mode }}",
+              "prerelease": "${{ inputs.prerelease }}"
             }
 
       - name: Build OS image
-        if: ${{ fromJSON(github.event.inputs.automatic_mode) == true }}
+        if: ${{ fromJSON(inputs.automatic_mode) == true }}
```

actionlint (1.7.4)

1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)